Realm/Realm.Unity/Editor/Common/RealmWeaver.Schema.cs

```diff
@@ -119,6 +119,14 @@ private MethodDefinition GetModuleInitializer()
                 HasThis = false
             };
 
+#if UNITY_IOS || ENABLE_IL2CPP
```

Comment:
Instead of doing preprocessor directives, we should always check for a reference to UnityEngine, but that check should move to ImportedReferences.